src/Int_ascending.java

```diff
@@ -0,0 +1,18 @@
+public class Int_ascending {
+
+	
+	public static void ascending(int[]arr) {
```

Comment:
This is great code Andy! Just so you have a comment/change to make for the assignment you could add a header to your ascending method to explain what it does and what arguments it takes